README.md

```diff
@@ -75,6 +76,17 @@ RECAPTCHA3_SECRET=my_secret
 Usage
 -----
 
+### How to use reCAPTCHA globally (meaning even in China):
+
+Use `'recaptcha.net'` host in your code when `'google.com'` is not accessible.
+
+```yaml
+# config/packages/karser_recaptcha3.yaml (or app/config/config.yml if using Symfony3)
+
+karser_recaptcha3:
+    host: 'recaptcha.net' # default is 'google.com'
```

Comment:
Should be `host: 'www.recaptcha.net' # default is 'www.google.com'`

Reply:
Yes, I forgot it 😉